dbptk-modules/dbptk-module-siard/src/main/java/com/databasepreservation/modules/siard/in/metadata/typeConverter/SQLStandardDatatypeImporter.java

```diff
@@ -163,8 +163,13 @@ protected Type getNationalCharType(String typeName, int columnSize, int decimalD
   @Override
   protected Type getCharType(String typeName, int columnSize, int decimalDigits, int numPrecRadix) {
     SimpleTypeString type = new SimpleTypeString(columnSize, false);
-    type.setSql99TypeName("CHARACTER", columnSize);
-    type.setSql2008TypeName("CHARACTER", columnSize);
+    if (columnSize == 0) {
+      type.setSql99TypeName("CHARACTER");
+      type.setSql2008TypeName("CHARACTER");
+    } else {
+      type.setSql99TypeName("CHARACTER", columnSize);
+      type.setSql2008TypeName("CHARACTER", columnSize);
+    }
```

Comment:
This does not seem to be part of this PR, please create a new PR and address which issue this modification belongs to.

Reply:
Issue created https://github.com/keeps/dbptk-developer/issues/627